test/celeritas/ext/RootImporter.test.cc

```diff
@@ -12,6 +12,8 @@
 
 #include "corecel/Types.hh"
 #include "corecel/cont/Range.hh"
+#include "celeritas/ext/RootImporter.hh"
```

Comment:
This include is already at the top of the file: the header being tested is always the first include in a test (to avoid hiding missing includes).